Environment/AeroAllergenObserved/doc/spec.md

```diff
@@ -0,0 +1,172 @@
+    + Normative References: [https://schema.org/address](https://schema.org/address)
+    + Mandatory if `location` is not present. 
+    
++ `dateObserved` : The date and time of this observation in ISO8601 UTCformat. It can be represented by an specific time instant or by an ISO8601 interval. 
+    + Attribute type: [DateTime](https://schema.org/DateTime) or an ISO8601 interval represented as [Text](https://schema.org/Text). 
+    + Mandatory
+    
++ `source` : A sequence of characters giving the source of the entity data.
+    + Attribute type: [Text](https://schema.org/Text) or [URL](https://schema.org/URL)
+    + Optional
+    
++ `allergenRisk` : Overall allergen risk corresponding to the aero allergens observed.
+    + Attribute type: [Text](https://schema.org/Text)
+    + Example values defined by the [European Aeroallergen Network](https://www.ean-net.org/en/):
+  (`none`, `low`, `moderate`, `high`, `very high`).
+  As this can be different between countries, regulations or implementations, the set of allowed values will depend on the reference specification used.
+  It is recommended that implementations use the same naming conventions as exemplified above (lower case starting words, camel case when compound terms are used)
+    + Attribute metadata:
+        + `referenceSpecification` : Specification that must be taken as reference when interpreting the supplied qualitative value. 
+            + Type: [Text](https://schema.org/Text) or [URL](https://schema.org/URL)
+            + Mandatory
+    + Optional
+  
++ `refDevice` : A reference to the device(s) which captured this observation.
+    + Attribute type: Reference to an entity of type `Device`
+    + Optional
+
+### Representing aero allergens concentration
+
+In order to enable a proper management of the the different aero allergens concentration,
+*for each* aero allergens we have to use an attribute that *MUST* refers exactly
+to the conventional name of the allergen (usually the latin name of
+the associated plant) concatenated with the string `Concentration`,
+e.g. `alnusConcentration` to measure
+the concentration (usually in grains per cubic meter) of 
+[alder](https://en.wikipedia.org/wiki/Alder) pollen.
+
+The structure of such an attribute will be as follows:
+
++ Attribute name: Equal to the name of the allergen plus the suffix `Level`, for instance `alnusConcentration`.
+
++ Attribute type: [Number](https://schema.org/Number)
+
++ Attribute value: corresponds to the concentration of the allergen as a number.
+
++ Attribute metadata:
+    + `timestamp` : optional timestamp for the observed value in ISO8601 format.
+        It can be omitted if the observation time is the same as the one captured by the `dateObserved` attribute at entity level.
+        + Type: [DateTime](https://schema.org/DateTime)
+    + `unitCode` : The unit code (text) of measured concentration (usually grains per cubic meter).
```

Comment:
is there a unit code for this measurement? can you put it as an example?

Reply:
gr/m3, added in in the text